geometry_msgs/PoseStamped[] data
